horace_core/sqw/@sqw/sqw.m

```diff
@@ -89,13 +89,16 @@
         
         % Public getters/setters expose all wrapped data attributes
         function val = get.data(obj)
-            val = '';
-            if ~isempty(obj.data_)
-                val = obj.data_;
-            end
+            val = obj.data_;
         end
         function obj = set.data(obj, d)
-            obj.data_ = d;
+            %if isa(d,'data_sqw_dnd') || isempty(d)
```

Comment:
unclear why you have commented out all the conditional code so the executable code is still the same. What are you intending to do with this?

Reply:
it should be such check, but apparently if its enabled, tests are failing as it sometimes gets a structure.
This should not happen, but fixing this is out of scope of this ticket after all.  Left here for the future -- structure should not appear here.